tests/AfricasTalkingTest.php

```diff
@@ -4,6 +4,7 @@
 use AfricasTalking\SDK\AfricasTalking;
 use GuzzleHttp\Exception\GuzzleException;
 
+#[\AllowDynamicProperties]
```

Comment:
If #46 is merged, This will not be auto-loaded in user-land and Dynamic properties should be OK in tests for now